README.adoc

```diff
@@ -28,7 +28,7 @@ Changes to already released versions of the documentation must be merged to a `m
 * `DC-caasp-*`: Configuration files for the exported guides.
 * `adoc/`: Contains all the pages that make up the content.
 * `adoc/book_*`: Meta files collating pages into a guide document.
-* `adoc/attributes.adoc` - Contains all version numbers of the product and it's components.
+* `adoc/attributes.adoc` - Contains all version numbers of the product and its components.
```

Comment:
This file isn't a complete list. It is an expansion on the `entities.adoc` file. A full list of components can be found in the release project. We're working on automatically adding the component versions to the attribute file but this still will only list the components we've chosen by hand to be synced because they need to updated so much.

Maybe in the long run it would make sense to dump nearly all the component versions into the docs attributes file to be able to reference most of them when needed. Right now this wasn't necessary.

We should not communicate this file as anything that can be used to get a list of component versions for users. It's too incomplete to actually function like that.